backend/canisters/online_users/api/src/updates/c2c_remove_user.rs

```diff
@@ -0,0 +1,12 @@
+use candid::{CandidType, Principal};
+use serde::{Deserialize, Serialize};
+
+#[derive(CandidType, Serialize, Deserialize, Debug)]
+pub struct Args {
+    pub principal: Principal,
```

Comment:
You don't want to batch them?

Reply:
This happens so infrequently that it doesn't make sense to batch them. I won't be pushing backdated ones.